src/VisualStudio/Core/Impl/SolutionExplorer/DiagnosticItem/BaseDiagnosticItem.BrowseObject.cs

```diff
@@ -56,7 +53,7 @@ public string HelpLink
             {
                 get
                 {
-                    return _diagnosticItem.GetHelpLink()?.AbsoluteUri;
+                    return _diagnosticItem.GetHelpLink()?.AbsoluteUri!;
```

Comment:
Interesting syntax. Should this be `_diagnosticItem.GetHelpLink().AbsoluteUri!` if we know the result of the expression cannot be null?

Reply:
Alright I must have misread something here -- I had thought I was suppressing AbsoluteUri being null but clearly the ? doesn't make sense either. I'll...inspect further. :smile:

Reply:
Alright I couldn't read a tooltip. Fixed.